.github/workflows/release_subctl_devel.yml

```
@@ -17,28 +17,26 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Fetch all git tags
-        run: git fetch origin +refs/tags/*:refs/tags/*
-
       - name: Generate subctl version
-        run: echo "SUBCTL_TAG=${GITHUB_REF##*/}" >> $GITHUB_ENV
+        run: |
+          echo "SUBCTL_TAG=${GITHUB_REF##*/}" >> $GITHUB_ENV
+          echo "RELEASE=release-${GITHUB_REF##*/}" >> $GITHUB_ENV
 
       - name: Generate the subctl release artifacts
+        run: make build-cross VERSION=${{ env.SUBCTL_TAG }}
+
+      - name: Install GH CLI
         run: |
-          make build-cross VERSION=${{ env.SUBCTL_TAG }}
-          echo "BINARIES=$(find dist/ -type f -name '*.tar.xz' -printf '%p ')" >> $GITHUB_ENV
+          curl -sLo /tmp/gh.deb https://github.com/cli/cli/releases/download/v1.11.0/gh_1.11.0_linux_amd64.deb
+          sudo dpkg -i /tmp/gh.deb
```

```suggestion
          sudo apt install /tmp/gh.deb
```

to deal with dependencies if that’s ever necessary.